packages/datadog-instrumentations/src/router.js

```diff
@@ -170,7 +170,7 @@ function createWrapRouterMethod (name) {
 
 const wrapRouterMethod = createWrapRouterMethod('router')
 
-addHook({ name: 'router', versions: ['>=1'] }, Router => {
+addHook({ name: 'router', versions: ['>=1 <2.0.0'] }, Router => {
```

Comment:
So just for my own clarity

We don't support Express v5+ _because_ we actually don't support Router v2+.
Which means that we first need to support v2 of Router and then we can see if that fixes Express v5, if not then we need to also support Express v5

Reply:
Yes, we'd need to properly support Router 2.x along with Express 5.x. In my local testing, I got spans from `express.request` with Express 5.x if we don't try to trace `router 2`, but not any of the express middleware spans. To fully support Express 5.x, we'd need to create middleware spans too.